packages/battery/ios/battery.podspec

```diff
@@ -12,8 +12,8 @@ Flutter plugin for accessing information about the battery.
   s.license          = { :file => '../LICENSE' }
   s.author           = { 'Flutter Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :path => '.' }
-  s.source_files = 'Classes/**/*'
```

Comment:
**/* isn't really right, either, since now all the files here (and in sub-directories) are considered source (podspec, Assets/, etc.).

I'd prefer keeping the source code in a separate directory, but we can rename it so 'src' or something else, if someone doesn't like 'Classes'.